Packs/GitLab/Integrations/GitLab/GitLab.yml

```diff
@@ -0,0 +1,3222 @@
+      description: The commit's parent ids.
+      type: String
+    - contextPath: GitLab.Commit.web_url
+      description: The commit's web url.
+      type: String
+  - name: gitlab-merge-request-list
+    description: Get all merge requests for this project.
+    arguments:
+    - auto: PREDEFINED
+      name: state
+      description: Return all merge requests or just those that are opened, closed, locked, or merged.
+      required: false
+      predefined:
+      - opened
+      - closed
+      - locked
+      - merged
+    - auto: PREDEFINED
+      name: order_by
+      defaultValue: created_at
+      description: Return requests ordered by created_at, title or updated_at fields. Default is created_at.
+      required: false
+      predefined:
+      - created_at
+      - updated_at
+      - title
+    - auto: PREDEFINED
+      name: sort
+      defaultValue: desc
+      description: Return requests sorted in asc or desc order. Default is desc.
+      required: false
+      isArray: false
+      predefined:
+      - asc
+      - desc
+    - name: milestone
+      description: Return merge requests for a specific milestone. None returns merge requests with no milestone. Any returns merge requests that have an assigned milestone.
+      required: false
+      isArray: false
+    - name: labels
+      description: Return merge requests matching a comma-separated list of labels. None lists all merge requests with no labels. Any lists all merge requests with at least one label. Predefined names are case-insensitive.
+      required: false
+      isArray: false
+    - name: created_after
+      description: Return merge requests created on or after the given time. Expected in ISO 8601 format.
+      required: false
+      isArray: false
+    - name: created_before
+      description: Return merge requests created on or before the given time. Expected in ISO 8601 format.
+      required: false
+      isArray: false
+    - name: updated_after
+      description: Return merge requests updated on or after the given time. Expected in ISO 8601 format.
+      required: false
+      isArray: false
+    - name: updated_before
+      description: Return merge requests updated on or before the given time. Expected in ISO 8601 format.
+      required: false
+      isArray: false
+    - auto: PREDEFINED
+      name: scope
+      description: Return merge requests for the given scope- created_by_me, assigned_to_me, or all.
+      required: false
+      isArray: false
+      predefined:
+      - created_by_me
+      - assigned_to_me
+      - all
+    - name: author_id
+      description: Returns merge requests created by the given user id. Mutually exclusive with author_username.
+      required: false
+      isArray: false
+    - name: author_username
+      description: Returns merge requests created by the given username. Mutually exclusive with author_id.
+      required: false
+      isArray: false
+    - name: assignee_id
```

Comment:
How can we get the assignee_id or the reviewer_id?

Reply:
there is a api command - GET /users but it is not in the design

Reply:
There is no other way to get this? in the commands we have?
If not, let's talk to @Moishymat